python/tskit/ms.py

```diff
@@ -0,0 +1,129 @@
+#
+# MIT License
+#
+# Copyright (c) 2019 Tskit Developers
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+"""
+Convert tree sequences to ms output
+"""
+import numpy as np
+import sys
+import os
+
+from . import provenance
+
+class msWriter:
+    """
+    Writes a ms representation of the genotypes tree sequence to a
+    file-like object.
+    """
+
+    def __init__(
+        self,
+        tree_sequence,
+        print_trees=False,
+        precision=3,
+        mutation_rate=0,
+        recombination_rate=0,
+        migration_rate=0,
+        num_loci=1,
+        num_replicates=1,
```

Comment:
Hm - looks like you don't need most of these options. Do you need any of them? Other than num_replicates, maybe?